src/main/resources/db/migration/V1_5__insert_additional_facilities.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE Additional_Facilities (
+CategoryKey varchar(64) not null,
+ServiceID varchar(16),
+Key varchar(64) not null,
+Value_EN varchar(128) not null,
+Value_CY varchar(128),
+HintText_EN varchar(512),
+HintText_CY varchar(512),
+Lov_Order BIGINT,
+ParentCategory varchar(64),
+ParentKey varchar(64),
+Active varchar(1)
+);
+
+INSERT INTO additional_facilities (categorykey,serviceid,"key",value_en,value_cy,hinttext_en,hinttext_cy,lov_order,parentcategory,parentkey,active) VALUES
+	 ('AdditionalFacilities',NULL,'AF-IDC','Immigration Detention Centre',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-ICC','In Camera Court',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-SSC','Same Sex Courtroom',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-SD','Secure Dock',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-WS','Witness Screen',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-WR','Witness Room',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-VC','Video Conferencing',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-VF','Video Facility',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('AdditionalFacilities',NULL,'AF-PVL','Prison Video Link',NULL,NULL,NULL,NULL,NULL,NULL,'Y'),
+	 ('InterpreterLanguage',NULL,'bel-bel','Belorussian',NULL,NULL,NULL,NULL,NULL,NULL,'Y');
```

Comment:
This should be updated into interpreter_and_sign_language table

Reply:
Done Sujeeth Thank you so much ...

Reply:
Sorry @sahitya-desireddy - we already have this data inserted in here - https://github.com/hmcts/rd-commondata-api/blob/master/src/main/resources/db/migration/V1_3__insert_hearing_channel.sql#L1443. But that data has a typo. So we should update that record from `Belorussia` to `Belorussian`. So we need update script instead of insert